doc/wg/core/notes/core-notes-2025-01-15.md

```diff
@@ -0,0 +1,109 @@
+# Tock Meeting Notes 2025-01-15
+
+## Attendees
+- Branden Ghena
+- Leon Schuermann
+- Pat Pannuto
+- Hudson Ayers
+- Johnathan Van Why
+- Kat Fox
+- Brad Campbell
+- Alexandru Radovici
+- Chris Frantz
+
+## Updates
+ * Leon: Alistair asked to add a TickV release tag to the Tock repo, which I added. That tag coincides with the Tock 2.2 release and marks TickV 2.0.0 release within Tock.
+ * Branden: After a hiatus for holidays and conferences, we did a walk-through of all the Tock networking interfaces, what their state is, what their goals are, and what the blockers are. That might be interesting for others to look at, so see the notes from that call.
+ * Branden: https://github.com/tock/tock/pull/4307
+ * Hudson: What was the status of 15.4 for the release?
+ * Leon: UDP remains broken, and has likely been broken for a few years now. 15.4 raw is still working.
+ * Hudson: If I set aside time, is there interest in UDP in 15.4? Or is that deprecated?
+ * Leon: I think either is good. It should probably be fixed or officially deprecated though. You might check in with Tyler about it
+ * Hudson: I'll put that on my list then
+
+
+## Syscall Drivers Testability
+ * https://github.com/tock/tock/issues/4303
+ * Alex: We need a testing framework for capsules and need to see what the capsule returns.
+ * Alex: We need unit tests for sure. A problem with them is we can simulate things but we can't extract the CommandReturn. The proper function for transforming it is within the kernel crate. So we can't use it without the kernel
```

Comment:
"can't extract the CommandReturn" -> "can't extract the CommandReturn's contents".